huaweicloud/services/vpc/data_source_huaweicloud_networking_secgroups.go

```diff
@@ -83,13 +94,13 @@ func getSecGroupDetail(secGroup groups.SecurityGroup) map[string]interface{} {
 	}
 }
 
-func filterAvailableSecGroups(d *schema.ResourceData, secGroups []groups.SecurityGroup) ([]map[string]interface{},
+func filterAvailableV1SecGroups(d *schema.ResourceData, secGroups []v1Groups.SecurityGroup) ([]map[string]interface{},
```

Comment:
please consider merge filterAvailableV1SecGroups and filterAvailableV3SecGroups into one func

Reply:
The current codes is clearer.
Not adopted